static_src/stores/activity_store.js

```diff
@@ -66,35 +66,43 @@ class ActivityStore extends BaseStore {
     let activity;
     switch (action.type) {
       case activityActionTypes.EVENTS_FETCH:
+        this.fetching = true;
+        this.fetched = false;
         cfApi.fetchSpaceEvents(action.spaceGuid);
+        this.emitChange();
         break;
 
       case activityActionTypes.EVENTS_RECEIVED:
+        this.fetching = false;
+        this.fetched = true;
```

Comment:
Yea I did think of this. Most other stores that are listening for more then one set of data, I decide what data is most important, but in this case, they're on equal footing. I think I'd prefer to get something out for now, but will try testing it out.